code/ATMOSPHERICS/components/omni_devices/omni_base.dm

```diff
@@ -48,11 +48,11 @@
 
 /obj/machinery/atmospherics/omni/update_icon()
 	if(stat & NOPOWER)
-		copy_overlays(overlays_off, TRUE)
+		overlays = overlays_off
```

Comment:
These ones are still using the broken systme of overlays

Reply:
fixed now, all overlays on file equal to the original